OwinOAuthProvidersDemo/App_Start/Startup.Auth.cs

```diff
@@ -42,6 +44,8 @@ public void ConfigureAuth(IAppBuilder app)
             //app.UseTripItAuthentication("", "");
 
             //app.UseGitHubAuthentication("", "");
+            app.UseGitHubAuthentication("github client_id here", "github client_secret here");
+            app.UseFreeConferenceCallAuthentication("fcc client_id here", "fcc client_secret here");
```

Comment:
Update with actual client_id and client_secret (not to be shared or included in the PR)